pset-iterators.js

```diff
@@ -65,7 +73,13 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 2:')
 
 // Add your code below this line
+const presidentsNamedJames = presidents.filter(function (president) {
+  name=president.president.split(" ")
+  return name[0] === 'James'
+})
 
+
+console.log(presidentsNamedJames)
```

Comment:
@AtleLohrmann Good job using the filter method to return all of the presidents whose name is James. Good job using the split method to return an array with both first and last names separated.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L62)